src/github/client/errors.ts

```diff
@@ -1,13 +1,16 @@
-import { AxiosError } from "axios";
+import {AxiosError, AxiosResponse} from "axios";
 
 export class GithubClientError extends Error {
 	status?: number;
-	cause: AxiosError;
-	constructor(message: string, cause: AxiosError, status?: number) {
+	cause?: AxiosError;
+
+	constructor(message: string, status?: number, cause?: AxiosError) {
```

Comment:
I've swapped "status" and "cause", because for GraphQL errors don't have axios exceptions, but there is status code (200)